PinpointKit/PinpointKit/Sources/ASLLogger.m

```diff
@@ -9,17 +9,54 @@
 #import "ASLLogger.h"
 #import <asl.h>
 
+@interface ASLLogger ()
+
+@property (nonatomic, nullable) NSString *bundleIdentifier;
+@property (nonatomic, nullable) NSString *senderName;
+@property (nonatomic) NSDate *logDate;
+
+@end
+
 @implementation ASLLogger
 
+- (instancetype)init {
+    self = [super init];
+    
+    _logDate = [NSDate date];
+    
+    return self;
+}
+
+- (instancetype)initWithBundleIdentifier:(NSString *)bundleIdentifier {
```

Comment:
I forget exactly, but do we have to mark these two as designated initializers? I think that to enforce their designatedness they have to call `super init`, in which case you'd want to set the log date in a new common initialization function that both call.